### PR TITLE
New package: newsraft-0.17

### PR DESCRIPTION
**Pull request by BryceVandegrift**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

**Review by daniel-eys — APPROVED** [review]

**Comment by BryceVandegrift** [comment]
It seems to be taking a while to get this package merged.

**Comment by daniel-eys** [comment]
Just in case a void dev comes by: I'm using this for a while now and it works great! So ready to merge

File: srcpkgs/newsraft/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+version=0.17
+revision=1
+build_style=gnu-makefile
+make_install_args="EXAMPLES_DIR=${DESTDIR}/usr/share/examples/newsraft"
+hostmakedepends="scdoc"
+makedepends="libcurl-devel expat-devel gumbo-parser-devel ncurses-devel sqlite-devel yajl-devel"
+short_desc="Feed reader with ncurses user interface"
+maintainer="Bryce Vandegrift <bryce@brycevandegrift.xyz>"
+license="ISC"
+homepage="https://codeberg.org/grisha/newsraft"
```
Comment (mhmdanas) [inline srcpkgs/newsraft/template:12]:
```suggestion
homepage="https://codeberg.org/grisha/newsraft"
changelog="https://codeberg.org/grisha/newsraft/releases"
```

**Review by mhmdanas — APPROVED** [review]
Tested this locally, seems to be working fine. Note that I'm not particularly interested in this package, the approval is just to show that this seems good to me on the technical side of things.

**Comment by xythh** [comment]
This has been working for me as well on x86_64-glibc, updated to version 0.19 in my localrepo.

**Pull request merged**